src/Microsoft.Azure.WebJobs.Extension.Storage/WebJobs.Extension.Storage.csproj

```diff
@@ -0,0 +1,90 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Microsoft.Azure.WebJobs.Shared\WebJobs.Shared.projitems" Label="Shared" />
+
+  <Import Project="..\Microsoft.Azure.WebJobs.Shared.Storage\Microsoft.Azure.WebJobs.Shared.Storage.projitems" Label="Shared" />
+  <Import Project="..\..\build\common.props" />  
+  
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Can you add the correct default namespace Microsoft.Azure.WebJobs.Extensions.Storage - it's defaulting to WebJobs.Extension.Storage